src/main/java/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,30 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeleteHistories {
+    private final List<DeleteHistory> deleteHistories;
+
+    public DeleteHistories() {
+        this.deleteHistories = new ArrayList<DeleteHistory>();
+    }
```

Comment:
DeleteHistories를 불변 객체로 사용해보는 것은 어떨까요?
Collection에서 제공하는 unmodifiableList는 리스트에 대한 추가, 삭제를 막아주는 기능으로 일급 컬렉션도 좀 더 불변에 가까워질 수 있습니다 :)
```suggestion
    public DeleteHistories(List<DeleteHistory> deleteHistories) {
        this.deleteHistories = Collections.unmodifiableList(deleteHistories);
    }
```

Reply:
일급 컬렉션을 불변객체로 만들생각을 못해봤는데 앞으로는 고려해서 설계하겠습니다!
DeleteHistories는 제안해주신대로 수정했습니다~! 😊